src/main/webapp/WEB-INF/jsp/viewNews.jsp

```diff
@@ -110,16 +110,18 @@
                         <h3 class="title">{{title}}</h3>
                     </div>
                 </div>
-                <div class="news-stories feed">
+                <ul class="news-stories feed">
                     {{{news_stories entries}}}
-                </div>
+                </ul>
         </script>
         <script type="text/template" id="${n}news-story-template">
             {{#each this}}
-                <h3>
-                    <a href="{{link}}" ${ newWindow ? "target=\"_blank\"" : "" }>{{title}}</a>
-                </h3>
-                <p>{{description}}</p>
+                <li>
+					<h3>
+						<a href="{{link}}" ${ newWindow ? "target=\"_blank\"" : "" }>{{title}}</a>
+					</h3>
+					<p>{{description}}</p>
+				</li>
             {{/each}}
         </script>
 
```

Comment:
Oh I should have checked other pull requests for this portlet. I created a Jira and fixed this issue with a few lines of CSS.